webui/react/src/services/decoder.ts

```diff
@@ -40,17 +40,16 @@ export const jsonToLogin = (data: unknown): LoginResponse => {
   return { token: io.token };
 };
 
-export const jsonToDeterminedInfo = (data: unknown): types.DeterminedInfo => {
-  const io = ioTypes.decode<ioTypes.ioTypeDeterminedInfo>(ioTypes.ioDeterminedInfo, data);
+export const jsonToDeterminedInfo = (data: Sdk.V1GetMasterResponse): types.DeterminedInfo => {
   return {
-    clusterId: io.cluster_id,
-    clusterName: io.cluster_name,
-    masterId: io.master_id,
+    clusterId: data.clusterId,
+    clusterName: data.clusterName,
+    masterId: data.masterId,
     telemetry: {
-      enabled: io.telemetry.enabled,
-      segmentKey: io.telemetry.segment_key || undefined,
+      enabled: data.telemetryEnabled === true,
+      segmentKey: undefined,
```

Comment:
**blocking:** The key is required to send telemetry data over to sement.

Reply:
Should be good now.